modules/ROOT/pages/dataweave-system-properties.adoc

```diff
@@ -1,55 +1,31 @@
 = DataWeave System Properties
-
-
-DataWeave supports several system properties. To use these properties in application development and deployments, see xref:mule-runtime::mule-app-properties-system.adoc[System Properties].
-
+DataWeave supports several system properties. To use these properties in application development and deployments, see xref:mule-runtime:mule-app-properties-system.adoc[System Properties].
 [%header%autowidth.spread,cols=",a"]
 |===
 |Property |Description
-
-//com.mulesoft.dw.buffer.memory.monitoring//
-|`com.mulesoft.dw.buffer.memory.monitoring`|
-_Experimental:_ When set to `true`, logs a message each time a slot from the memory pool is taken or released. The behavior is useful for troubleshooting.
-`com.mulesoft.dw.buffer.memory.monitoring` supports an _experimental feature_ that is subject to change or removal from future versions of DataWeave.
-
-* Type: `Boolean`
-* Default: `false`
-
-
-//com.mulesoft.dw.buffered_char_sequence.enabled
+//com.mulesoft.dw.buffered_char_sequence.enabled//
 |`com.mulesoft.dw.buffered_char_sequence.enabled`|
-Controls whether an internal `BufferedCharSequence` improvement is enabled or disabled. When this property is set to `true`, DataWeave avoids loading large fields (greater than 1.5M) into memory. These fields are read in chunks. This feature reduces performance to prevent an out-of-memory error.
-
-
+Controls whether an internal BufferedCharSequence improvement is enabled or disabled. When this property is set to `true`, DataWeaves avoid load large fields (bigger than 1.5M) into memory. This fields are read by chunk. This feature introduces a performance reduction preventing an Out of Memory error.
 * Type: `Boolean`
 * Default: `true`
-
-
 //com.mulesoft.dw.buffersize//
 |`com.mulesoft.dw.buffersize`|
 Sets the size (in bytes) of in-memory input and output buffers that DataWeave uses to retain processed inputs and outputs.
-DataWeave stores payloads that exceed this size in temporary files `dw-buffer-index-${count}.tmp` and `dw-buffer-output-${count}.tmp`.
-See xref:dataweave-memory-management.adoc[DataWeave Memory Management].
-
-* Type: `Number` (in bytes)
-* Default: `8192` bytes (8 KB)
-
-
+* Type: `Integer`
+* Default: `8192`
 //com.mulesoft.dw.coercionexception.verbose//
 |`com.mulesoft.dw.coercionexception.verbose`|
 By default, adds to coercion exceptions more information about data that fails to coerce. When this property is set to `false`, DataWeave does not display the additional metadata.
-
 * Type: `Boolean`
 * Default: `true`
-
-
+//com.mulesoft.dw.cpulimit.watchdog//
+|`com.mulesoft.dw.cpulimit.watchdog`|
+When set, script execution time is monitored and limited
```

Comment:
Needs more detailed. Where is is monitored? How do you discover that? And how/how much is the time limited?

Reply:
And what happens when you set it to false? Who uses this?

Reply:
TODO: ask on dw channel

Reply:
asked on channel

Reply:
edited in source, but still need more detail